pyanaconda/payload/__init__.py

```diff
@@ -1010,6 +1010,12 @@ def __init__(self, data):
     def preInstall(self):
         super().preInstall()
 
+        # Set up the FIPS mode in the target system before the package installation.
+        if flags.cmdline.get("fips") == "1":
+            src = "/etc/crypto-policies/back-ends/"
+            dst = os.path.normpath(util.getSysroot() + src)
+            shutil.copytree(src, dst, symlinks=True, dirs_exist_ok=True)
```

Comment:
I'd like to be a bit more defensive around here, in the lines of `with open('/etc/crypto-policies/config') as f: assert f.read().strip() == 'FIPS'`, maybe even with an extra `assert 'FIPS' in os.path.realpath('/etc/crypto-policies/back-ends/opensshserver.config')`.

Reply:
Fixed.